selenium_tests/course_info/course_info_base_test_case.py

```diff
@@ -55,3 +61,37 @@ def search_for_course(self, type=None, school=None, term=None, year=None,
             self._search_page.select_year(year)
         self._search_page.submit_search(search_term)
 
+    def _api_post(self, path=None, body=None):
```

Comment:
Just a suggestion for next time, these could all be one call that takes the method as an argument.